sonar-java-plugin/pom.xml

```diff
@@ -49,22 +49,11 @@
     </dependency>
 
     <dependency>
-      <groupId>org.codehaus.sonar</groupId>
-      <artifactId>sonar-java-api</artifactId>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.codehaus.sonar</groupId>
+      <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-plugin-api</artifactId>
       <scope>provided</scope>
     </dependency>
```

Comment:
@benzonico `slf4j-api` is still provided at runtime, even if it is not transitive dependency of `sonar-plugin-api`, so that you can reduce size of plugin by declaring:
```
    <dependency>
      <groupId>org.slf4j</groupId>
      <artifactId>slf4j-api</artifactId>
      <scope>provided</scope>
    </dependency>
```

Also you package `jcl-over-slf4j` - I don't know if this one is also provided at runtime, but maybe it is even not need for Java plugin and can be completely excluded.